contracts/bridge/contracts/interfaces/ILightClient.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.7.0;
+
+interface ILightClient {
+    function verify_messages_proof(
+        bytes32 laneDataHash,
```

Comment:
Request document, lane is combined of outbound and inbound. here the data is outbound or inbound, not together.